models/advanced-home-example/readme.md

```diff
@@ -0,0 +1,40 @@
+# Changes made between basic and advanced
+
+Semantic Types<br />
```

Comment:
You might want to put these in a bulleted list, and link each one to the heading below so it's like a little TOC